.github/ISSUE_TEMPLATE/3_technical_task.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: 🔨 Technical task
-about: Record an action that needs to be taken
+about: Task that is not necessarily user facing
```

Comment:
Hmm, neither's great. These tickets are things that have been requested somewhere else. 🤔

Reply:
Maybe I'm reading too much into this kind of ticket? Is it the one that is supposed to be the 90% of tickets around here or just a corner case?

Reply:
More like 99% at the moment!

Reply:
So who has requested this 99% of tickets somewhere else? To my understanding they are autonomously created as part of implementing a feature. We keep track of them but they are not necessarily clearly mapping to a functional requirement (e.g. "write this document", "refactor SASS unit testing", etc.)